### PR TITLE
Updated TiDB Roadmap 

### PR DESCRIPTION
**Pull request by Frank945946**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Updated TtiDB Roadmap 

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v8.5 (TiDB 8.5 versions)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-roadmap.md [hunk 1]
```diff
@@ -9,233 +9,252 @@ TiDB 路线图展示了 TiDB 未来的计划。随着我们发布长期稳定版
...
       <th>类别</th>
-      <th>2024 年底 LTS 版本</th>
-      <th>2025 年中 LTS 版本</th>
+      <th>2024 年底版本</th>
+      <th>2025 年中版本</th>
       <th>未来版本</th>
     </tr>
   </thead>
-  <tbody valign="top">
+  <tbody>
```
Comment (hfxsd) [inline tidb-roadmap.md:29]:
```suggestion
  <tbody valign="top">
```

File: tidb-roadmap.md [hunk 2]
```diff
@@ -9,234 +9,295 @@ TiDB 路线图展示了 TiDB 未来的计划。随着我们发布长期稳定版
 
 在开发过程中，路线图可能会根据用户需求和反馈进行调整，请不要根据路线图的内容制定上线计划。如果你有功能需求，或者想提高某个特性的优先级，请在 [GitHub](https://github.com/pingcap/tidb/issues) 上提交 issue。
 
+> **注意：**
+> 
+> - 没有注明 GA (Generally Available) 的特性，均为实验特性。
```
Comment (hfxsd) [inline tidb-roadmap.md:14]:
```suggestion
> - 未注明 GA (Generally Available) 的特性，均为实验特性。
```

File: tidb-roadmap.md [hunk 3]
```diff
@@ -9,234 +9,295 @@ TiDB 路线图展示了 TiDB 未来的计划。随着我们发布长期稳定版
...
+      </td>
+      <td>
+        <ul>
+          <li>
+            <b>TiKV 数据缓存 </b>
+            <br />TiKV 在内存中维护数据的最近版本，减少对多版本数据的反复扫描，进而提升性能。
+          </li>
+          <br />
+          <li>
+            <b>自动配置统计信息收集的并行度(GA)</b>
```
Comment (hfxsd) [inline tidb-roadmap.md:43]:
```suggestion
            <b>自动配置统计信息收集的并行度 (GA)</b>
```

File: tidb-roadmap.md [hunk 4]
```diff
@@ -9,234 +9,295 @@ TiDB 路线图展示了 TiDB 未来的计划。随着我们发布长期稳定版
...
+            <br />TiDB 根据部署的节点数以及硬件规格自动设置统计信息收集的任务并行度和扫描并发度，提升收集速度。
+          </li>
+          <br />
+          <li>
+            <b>加速数据库恢复</b>
+            <br />缩短全量数据库恢复和 Point-in-time recovery (PITR) 所需的时间。
```
Comment (hfxsd) [inline tidb-roadmap.md:49]:
```suggestion
            <br />缩短全量数据库恢复和 Point in Time Recovery (PITR) 所需的时间。
```

**Review by Frank945946 — COMMENTED** [review]
LGTM

**Review by joey-yez — APPROVED** [review]

**Review by songrijie — APPROVED** [review]

File: tidb-roadmap.md [hunk 5]
```diff
@@ -9,234 +9,296 @@ TiDB 路线图展示了 TiDB 未来的计划。随着我们发布长期稳定版
...
-            当统计信息被更新后，用户可以查看统计信息的历史版本，并能够选择恢复到过去某个版本的统计信息。
+            <b>实例级执行计划缓存 (GA)</b>
+            <br />同一个 TiDB 实例的所有会话可以共享执行计划缓存，提升内存利用率。
           </li>
           <br />
           <li>
-            <b>更可靠的数据备份</b><br />
-            减少数据备份过程中可能出现的内存不足等问题，并确保备份数据的可用性。
+            <b>资源组优先满足限额内定义的用量 (RU) (GA)</b>
+            <br />动态管调整 Burstable 资源组使用的资源上限。在不影响其他资源组限额的情况下，充分利用剩余资源。
```
Comment (qiancai) [inline tidb-roadmap.md:216]:
```suggestion
            <br />动态调整 Burstable 资源组使用的资源上限。在不影响其他资源组限额的情况下，充分利用剩余资源。
```

File: tidb-roadmap.md [hunk 6]
```diff
@@ -9,234 +9,296 @@ TiDB 路线图展示了 TiDB 未来的计划。随着我们发布长期稳定版
...
           </li>
```
Comment (qiancai) [inline tidb-roadmap.md:217]: 需要说下什么是 Burstable 资源组吗

File: tidb-roadmap.md [hunk 7]
```diff
@@ -9,234 +9,296 @@ TiDB 路线图展示了 TiDB 未来的计划。随着我们发布长期稳定版
...
-          </li>
         </ul>
       </td>
       <td>
         <ul>
           <li>
-            <b>自适应资源组</b><br />
-            资源组根据过往的运行情况自动调整资源组的 RU 设定。
+            <b>自适应资源组</b>
+            <br />资源组根据过往的运行情况自动调整资源组的 RU 设定。
```
Comment (qiancai) [inline tidb-roadmap.md:225]:
```suggestion
            <br />支持根据过往的运行情况自动调整资源组的 RU 设定。
```

File: tidb-roadmap.md [hunk 8]
```diff
@@ -9,234 +9,296 @@ TiDB 路线图展示了 TiDB 未来的计划。随着我们发布长期稳定版
...
+          <li>
+            <b>强化的内存保护</b>
+            <br />TiDB 主动对所有模块的内存使用进行监控，阻止一切可能影响系统稳定性的内存操作。
           </li>
           <br />
           <li>
-            <b>强化内存保护</b><br />
-            TiDB 主动监控所有模块的内存使用情况，阻止可能影响系统稳定性的内存操作。 
+            <b>自动 SQL 绑定</b>
+            <br />通过对 SQL 运行指标的收集和分析，对一部分执行计划自动创建绑定，提升 TP 类系统的执行计划稳定性。
```
Comment (qiancai) [inline tidb-roadmap.md:235]:
```suggestion
            <br />通过对 SQL 运行指标的收集和分析，对一部分执行计划自动创建绑定，提升 OLTP 类系统的执行计划稳定性。
```

File: tidb-roadmap.md [hunk 9]
```diff
@@ -9,234 +9,296 @@ TiDB 路线图展示了 TiDB 未来的计划。随着我们发布长期稳定版
...
         <b>数据库管理与可观测性</b>
-        <br />增强数据库可管理性及其生态系统
+        <br />通过主动监控和管理，确保系统平稳运行。
       </td>
       <td>
         <ul>
           <li>
-            <b>可靠的查询终止</b><br />
-            正在运行中的 SQL 语句能够被立即终止，并从 TiDB 和 TiKV 中释放相应的资源。
+            <b>可靠地终止操作 (GA)</b>
```
Comment (qiancai) [inline tidb-roadmap.md:259]:
```suggestion
            <b>可靠的 SQL 终止操作 (GA)</b>
```

File: tidb-roadmap.md [hunk 10]
```diff
@@ -9,234 +9,296 @@ TiDB 路线图展示了 TiDB 未来的计划。随着我们发布长期稳定版
...
           </li>
           <br />
           <li>
-            <b>支持查看表或 SQL 与热点 Region 的关系</b>
-          </li>
-          <br />
-          <li>
-            <b><code>IMPORT INTO</code> 支持逻辑导入</b>
+            <b>增加对 TiDB 和 TiKV CPU 时间的观测 (GA)</b>
+            <br />在 statements 记录、慢日志中增加 TiDB 和 TiKV CPU 时间的指标，方便快速定位造成 TiDB 或者 TiKV CPU 飙升的语句。
```
Comment (qiancai) [inline tidb-roadmap.md:270]:
```suggestion
            <br />在 statements 记录、慢日志中增加 TiDB 和 TiKV CPU 时间的指标，方便快速定位造成 TiDB 或者 TiKV CPU 飙升的 SQL 语句。
```

File: tidb-roadmap.md [hunk 11]
```diff
@@ -9,234 +9,296 @@ TiDB 路线图展示了 TiDB 未来的计划。随着我们发布长期稳定版
...
           <br />
         </ul>
       </td>
       <td>
         <ul>
           <li>
-            <b>细粒度定制统计信息收集策略</b>
-            <br />用户可以修改特定表的统计信息收集策略，例如健康度和并行度。
+            <b>细粒度定制统计信息收集策略 (GA)</b>
+            <br />你可以为特定表格定制统计策略，调整健康度和并发度等参数。
```
Comment (qiancai) [inline tidb-roadmap.md:279]:
```suggestion
            <br />你可以为特定的表定制统计策略，调整健康度和并发度等参数。
```

File: tidb-roadmap.md [hunk 12]
```diff
@@ -9,234 +9,296 @@ TiDB 路线图展示了 TiDB 未来的计划。随着我们发布长期稳定版
...
-            <br />用户可以修改分区中列的类型，无论该列是否为分区键。
+            <b>标准时间模型 (GA)</b>
+            <br />对 SQL 的运行时间进行标准化定义，以此为基础定义数据库负载。通过观测 statements 记录、慢日志、聚合的集群指标，用户能够准确发现产生异常负载的节点及 SQL。
           </li>
           <br />
           <li>
-            <b>设置 <code>IMPORT INTO</code> 的冲突策略</b>
-            <br />用户可以为导入数据时出现的冲突设置解决策略，例如报错退出、忽略或替换。
+            <b>增加对 TiFlash CPU 时间的观测 (GA)</b>
+            <br />在 statements 记录、慢日志中增加 TiFlash CPU 时间的指标，方便快速定位造成 TiFlash CPU 飙升的语句。
```
Comment (qiancai) [inline tidb-roadmap.md:299]:
```suggestion
            <br />在 statements 记录、慢日志中增加 TiFlash CPU 时间的指标，方便快速定位造成 TiFlash CPU 飙升的 SQL 语句。
```

File: tidb-roadmap.md [hunk 13]
```diff
@@ -247,83 +309,95 @@ TiDB 路线图展示了 TiDB 未来的计划。随着我们发布长期稳定版
...
           <br />
         </ul>
       </td>
     </tr>
     <tr>
       <td>
-        <b>安全</b>
-        <br />增强数据安全与隐私保护
+        <b>全面的数据安全和隐私保护</b>
+        <br />保护敏感数据的强大安全措施，确保顶级保护、加密和符合不断发展的隐私法规
```
Comment (qiancai) [inline tidb-roadmap.md:322]:
```suggestion
        <br />强化保护敏感数据的安全措施，确保顶级保护、加密和符合不断发展的隐私法规
```

Comment (qiancai) [reply tidb-roadmap.md:322]: reason: 其他类别介绍也都是动词开头的

File: tidb-roadmap.md [hunk 14]
```diff
@@ -247,83 +309,95 @@ TiDB 路线图展示了 TiDB 未来的计划。随着我们发布长期稳定版
...
+          <br />
         </ul>
       </td>
       <td>
         <ul>
           <li>
-            <b>基于标签的访问控制</b>
-            <br />支持通过配置标签的方式，以标签形式对数据进行访问控制。
+            <b>基于标签的访问控制机制</b>
+            <br />支持通过配置标签的方式，通过标签形式对数据进行访问控制。
```
Comment (qiancai) [inline tidb-roadmap.md:386]:
```suggestion
            <br />支持为数据配置标签，然后通过标签形式对数据进行访问控制。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: tidb-roadmap.md [hunk 15]
```diff
@@ -9,234 +9,296 @@ TiDB 路线图展示了 TiDB 未来的计划。随着我们发布长期稳定版
...
-             <b>TiKV 数据缓存</b><br />
-            TiKV 在内存中维护数据的最近版本，以减少对多版本数据的重复扫描，进而提升性能。
+            <b>PD 的路由功能微服务化</b>
+            <br />实现路由服务（Region 元数据的访问、更新）在 PD 中的独立部署，路由服务完全改造为无状态服务（无强领导者）、易于扩展，避免 PD 成为集群资源瓶颈。
           </li>
           <br />
           <li>
-             <b>分区表全局索引</b><br />
+            <b>减少统计信息收集时的 I/O 消耗 (GA)</b>
+            <br />当抽取部分数据样本收集统计信息时，TiKV 上仅扫描样本，以减少统计信息收集所消耗的时间和资源。
```
Comment (qiancai) [inline tidb-roadmap.md:73]:
```suggestion
            <br />当收集统计信息时，允许 TiKV 上仅扫描部分数据样本，以减少统计信息收集所消耗的时间和资源。
```

Comment (qiancai) [reply tidb-roadmap.md:73]: reason: 英文里是 “Allows you to scan only a portion of data samples on TiKV, reducing time and resource consumption for statistics collection.“

File: tidb-roadmap.md [hunk 16]
```diff
@@ -9,234 +9,296 @@ TiDB 路线图展示了 TiDB 未来的计划。随着我们发布长期稳定版
...
-            <b>PD 的 heartbeat 微服务化</b><br />
-            PD 的 heartbeat 可以独立部署和单独扩展，避免 PD 成为集群的资源瓶颈。
+            <b>表级别的负载均衡</b>
+            <br />PD 根据每个表上各 Region 的负载决定数据的调度策略。
           </li>
           <br />
           <li>
-            <b>减少统计信息收集时的 I/O 消耗</b><br />
-            在进行统计信息收集时，可以选择在 TiKV 上仅扫描部分数据样本，以减少统计信息收集所消耗的时间和资源。
+            <b>处理大数据量的系统表</b>
```
Comment (qiancai) [inline tidb-roadmap.md:106]:
```suggestion
            <b>系统表性能优化</b>
```

Comment (qiancai) [reply tidb-roadmap.md:106]: to match "System Table Performance Optimization" in the Englis PR

File: tidb-roadmap.md [hunk 17]
```diff
@@ -9,234 +9,296 @@ TiDB 路线图展示了 TiDB 未来的计划。随着我们发布长期稳定版
...
+            <br />当系统表中存有大量数据时，提升查询系统表的查询性能。
           </li>
           <br />
           <li>
-            <b>移除将 Limit 算子下推到 TiKV 的已知限制</b><br />
+            <b>增增强 Region 元数据存储的可扩展性</b>
+            <br />将 Region 元数据存储从 PD 迁移到 TiKV，存储层可以轻松实现无限制的扩展。
```
Comment (hfxsd) [inline tidb-roadmap.md:112]:
```suggestion
            <br />拆分专用的无状态路由器服务（区域元数据读/写），并将区域元数据存储从 PD 迁移到 TiKV。 元数据存储层将轻松实现无限扩展。
```

File: tidb-roadmap.md (hunk 6 above)
Comment (Frank945946) [reply tidb-roadmap.md:217]: @qiancai 不建议加说明，加了就比较长了。可以加一个超链接让用户参考，但是我看了一下没有合适的超链接 https://docs.pingcap.com/zh/tidb/stable/information-schema-resource-groups#%E7%A4%BA%E4%BE%8B

Comment (qiancai) [reply tidb-roadmap.md:217]: OK

**Review by hfxsd — APPROVED** [review]

**Pull request merged**